Find a functionally equivalent code:com.android.internal.telephony.CallerInfoAsyncQuery.release:COMMENT
Method Modifier: private     internal    
Comment:/**
 * Releases the relevant data.
 */

@@ -1,5 +1,5 @@
 {
-    mHandler.mQueryContext = null;
+    mHandler.mContext = null;
     mHandler.mQueryUri = null;
     mHandler.mCallerInfo = null;
     mHandler = null;

